WinRT.Runtime/ComWrappersSupport.netstandard2.0.cs

```diff
@@ -120,25 +120,30 @@ public static T FindObject<T>(IntPtr thisPtr)
             where T : class =>
             (T)UnmanagedObject.FindObject<ComCallableWrapper>(thisPtr).ManagedObject;
 
-        public static IUnknownVftbl IUnknownVftbl { get; } = new IUnknownVftbl
+        public static unsafe IUnknownVftbl IUnknownVftbl { get; } = new IUnknownVftbl
         {
-            QueryInterface = Do_Abi_QueryInterface,
-            AddRef = Do_Abi_AddRef,
-            Release = Do_Abi_Release
+            QueryInterface = (delegate* stdcall<IntPtr, ref Guid, out IntPtr, int>)Marshal.GetFunctionPointerForDelegate(Abi_QueryInterface),
+            AddRef = (delegate* stdcall<IntPtr, uint>)Marshal.GetFunctionPointerForDelegate(Abi_AddRef),
+            Release = (delegate* stdcall<IntPtr, uint>)Marshal.GetFunctionPointerForDelegate(Abi_Release),
         };
         
         public static IntPtr AllocateVtableMemory(Type vtableType, int size) => Marshal.AllocCoTaskMem(size);
 
+        private delegate int QueryInterface(IntPtr pThis, ref Guid iid, out IntPtr ptr);
+        private static QueryInterface Abi_QueryInterface = Do_Abi_QueryInterface; 
         private static int Do_Abi_QueryInterface(IntPtr pThis, ref Guid iid, out IntPtr ptr)
```

Comment:
does this need:
#if !NETSTANDARD2_0
            [UnmanagedCallersOnly]
#endif

same for AddRef & Release?

Reply:
This file is .NET Standard 2.0 only, so not in this case.